test/katex-spec.js

```diff
@@ -2431,10 +2438,30 @@ describe("A macro expander", function() {
         compareParseTree("\\foo", "x", {"\\foo": " x"});
     });
 
-    it("should consume spaces after macro", function() {
+    it("should consume spaces after control-word macro", function() {
         compareParseTree("\\text{\\foo }", "\\text{x}", {"\\foo": "x"});
     });
 
+    it("should consume spaces after macro with \\relax", function() {
+        compareParseTree("\\text{\\foo }", "\\text{}", {"\\foo": "\\relax"});
+    });
+
+    it("should consume spaces after \\relax", function() {
+        compareParseTree("\\text{\\relax }", "\\text{}");
+    });
+
+    it("should consume spaces after control-word function", function() {
+        compareParseTree("\\text{\\KaTeX x}", "\\text{\\KaTeX\\relax x}");
```

Comment:
So these two things parse the same, but I thought control words weren't allowed inside `\text{}`.  Why is it important that these two parse the same?

Reply:
I wanted to make sure that `\text{\KaTeX x}` does not render a space.  Ideally I'd say that `\text{\KaTeX x}` renders the same as `\text{{\KaTeX}x}` but that generates another group... perhaps I should tweak the test to actually look for features in the parse tree.  (Control words like `\KaTeX` definitely work in text mode.)

Reply:
I got an error when I tried `\text{\KaTeX}` on the demo page so maybe it's just some control words don't work.